extensions/camel/camel-aws-s3/runtime/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Same here, it should be 4 spaces.

The parent pom is OK.

Reply:
Done